File: tks-examples/test_concat/test_concat.php
<?php

function microtime_float()
{
   list($usec, $sec) = explode(" ", microtime());
   return ((float)$usec + (float)$sec);
}

$t = microtime_float();

$s1 = "hallo, ";
$s2 = "welt...";
$s3 = "some ";
$s4 = "string";
$s5 = "concatenations";

$r = "";

$l = 0;

for($i=0; $i<10000000; $i++)
{
   $r = $s1 . $s2 . $s3 . $s4 . $s5;
   $l = strlen($r);
}

echo "l=" .$l ."\n";
echo "ms = " . ((microtime_float()-$t)*1000) . "\n";

?>

File: tks-examples/test_concat/test_concat.py
#!/usr/bin/python

import time


s1 = "hallo, "
s2 = "welt..."
s3 = "some "
s4 = "string"
s5 = "concatenations"

r = ""

t = time.time()
l = 0;

for i in range(10000000):
  r = s1 + s2 + s3 + s4 + s5;
  l = len(r);

print "l=" + str(l);
print str(1000*(time.time()-t)) + "ms";




